src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AzureAnalyzerVerifier.cs

```diff
@@ -17,7 +17,7 @@ public static class AzureAnalyzerVerifier<TAnalyzer> where TAnalyzer : Diagnosti
     {
         private static readonly ReferenceAssemblies DefaultReferenceAssemblies =
             ReferenceAssemblies.Default.AddPackages(ImmutableArray.Create(
-                new PackageIdentity("Azure.Core", "1.21.0"),
+                new PackageIdentity("Azure.Core", "1.26.0"),
```

Comment:
Would it make sense to make this 1.25.0 so we could merge and apply it sooner?

Reply:
`NullableResponse<T>` needs to exist for the check to work

Reply:
Got it, makes sense.